modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -34,30 +34,30 @@ With this update, you can use the `developerHub.flavor` field to identify whethe
 [source,yaml,subs="+quotes"]
 ----
 developerHub:
-  flavor: &lt;flavor&gt;;
+  flavor: <flavor>
 ----
 
 `flavor`::
 Identify the flavor of Backstage that is running. Default value: `rhdh`
 
 
 [id="feature-rhidp-4419"]
-== Ability to manage PVCs in RHDH Operator
-
-You can now mount directories from pre-created PersistentVolumeClaims (PVCs) using the `spec.application.extraFiles.pvcs` field, while configuring {product-very-short} Operator.
+== Ability to manage Persistent Volume Claim (PVCs) in {product-very-short} Operator
 
+You can now mount directories from pre-created PersistentVolumeClaims (PVCs) using the `spec.application.extraFiles.pvcs` field, while configuring RHDH Operator.
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/configuring/index#configuring-the-deployment[Configuring {product} deployment when using the Operator].
 
 [id="feature-rhidp-4805"]
 == Authenticating with {rhbk-brand-name}
 
-With this update, you can use {rhbk-brand-name} as an authentication provider. The Keycloak plugin will now support ingesting users and groups with {rhbk-brand-name}. For more details, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/26.0/html-single/getting_started_guide/index#getting-started-zip-start-red-hat-build-of-keycloak[Authentication with {rhbk-brand-name}].
+With this update, you can use {rhbk-brand-name} as an authentication provider. The Keycloak plugin will now support ingesting users and groups with {rhbk-brand-name}. For more details, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/authentication/index#assembly-authenticating-with-rhbk[Authenticating with {rhbk-brand-name}].
```

Comment:
release notes are generated using JIRA as the single source of truth. Please apply your changes in https://issues.redhat.com/browse/RHIDP-4805 then run this script:

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/release-1.4/modules/release-notes/single-source-release-notes.py